languageserver/integration/initialization.go

```diff
@@ -41,6 +41,9 @@ func (i *FlowIntegration) initialize(initializationOptions interface{}) error {
 		i.config.EmulatorAddr,
 		grpc.WithInsecure(),
 	)
+
+	i.emulatorState = byte(emulatorState)
```

Comment:
```suggestion
	i.emulatorState = EmulatorState(emulatorState)
```

Reply:
That was exactly the thing I was looking for!